libs/features/interlinks.js

```diff
@@ -23,7 +23,7 @@ export function checkAndAddMatch(matches, contender, maxMatches) {
  * @param {string} path The location of the keywords file to be used for interlinks.
  * @param {number} limit The maximum amount of keywords to fetch from the file.  Default is 1000.
  */
-export default async function interlink(path, limit = 1000) {
+export default async function interlink(config, path = `${config.locale.contentRoot}/keywords.json`, limit = 1000) {
   const articleBody = document.querySelector('main');
```

Comment:
Can't you avoid a lot of this hassle by simply passing in the path from utils? Then you don't have to:

1. Change every single test.
2. Pass in the config object to interlinks
3. Set a default path in the function

Reply:
I think we're going to need the config for later tasks: 
- https://jira.corp.adobe.com/browse/MWPW-121870
- https://jira.corp.adobe.com/browse/MWPW-121872

It's not necessary this task though. I'll change it